RELEASE-NOTES.md

```diff
@@ -7,17 +7,32 @@ The format of this file is based on [Keep a Changelog](http://keepachangelog.com
 
 The Axom project release numbers follow [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## [Unreleased] - Release date yyyy-mm-dd
+## [Version 0.3.2] - Release date 2019-09-22
```

Comment:
Please add a new stub section for ``Unreleased``
(unless we add that one when we merge master back to develop ?)

Reply:
I think we should do that on develop when the first bullet is added to the release notes. That way we don't have an empty stub section on the release branch..

Reply:
Seems reasonable.  

Adding it when we merge master back to develop might prevent developers from adding new notes to a previous release though.